src/aws_credentials.erl

```diff
@@ -116,13 +116,20 @@ format_status(_, [_PDict, State]) ->
 
 fetch_credentials() ->
     ShouldCatch = not application:get_env(aws_credentials, fail_if_unavailable, true),
+    ShouldDisplayST = application:get_env(aws_credentials, log_full_stacktrace_on_error, true),
```

Comment:
In #24 I am proposing to drop support for OTP 20. If that gets in, the Stacktrace will be logged instead of using the `error_logger` module and this won't probably be needed.

Reply:
@netrome We agreed to drop support for OTP 20 altogether, so this should not be necessary any longer, but thanks for this anyway :)

Reply:
Indeed. I'm closing this PR.